content/blog/webrtc-demo.md

```diff
@@ -0,0 +1,192 @@
+Title: Creating our WebRTC demo
+Date: 2021-10-29 16:30:00
+Author: Manon Gerray
+Category: Wazo Platform
+Tags: wazo platform, webrtc, front-end
+Slug: creating-our-webrtc-demo
+Status: published
+
+## Introduction
+
+Hello everyone! Six months ago I started a one year work/study training program at Wazo as a front-end developer. Wazo has a huge development framework to get used to, and what is better than a concrete project to learn the Wazo SDK? 
+
+That is how I have been assigned the Web RTC demonstration project, which will be useful to anyone wondering what we are able to do with the Wazo SDK. The main goals I had when I took over the project were the following : 
+* Totally redesign the application to improve user experience
+* Set up phone & video call, transfer & indirect transfer
+* Being able to call another number while already on call & switch between interlocutors
+
+## Launching the demo at home
+
+If you want to take a closer look at the code and the demo itself, you can go and set it up with these commands: 
+```
+apt install wazo-plugin-cli
+wazo-plugind-cli -c 'install git https://github.com/wazo-platform/wazo-webrtc-demo'
+```
+Otherwise, you can clone the following repository : [wazo-webrtc-demo](https://github.com/wazo-platform/wazo-webrtc-demo) and follow the instructions in the README file on the master branch. 
```

Comment:
```suggestion
Otherwise, you can clone the following repository : [wazo-webrtc-demo](https://github.com/wazo-platform/wazo-webrtc-demo) and follow the instructions in the [README.md](https://github.com/wazo-platform/wazo-webrtc-demo/blob/master/README.md). 
```